integration_test/third_party_apps_test/applications/couchbase/enable

```diff
@@ -13,13 +13,15 @@ fi
 sudo cp /etc/google-cloud-ops-agent/config.yaml /etc/google-cloud-ops-agent/config.yaml.bak
 
 # Configure the Ops Agent.
-sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << \EOF
 metrics:
   receivers:
     couchbase:
       type: couchbase
       username: admin
-      password: password
+      # Use plaintext password if you are not using the googlesecretmanager provider.
+      # password: password 
+      password: ${googlesecretmanager:projects/228437877067/secrets/ops-agent-3p-test-couchbase-password/versions/1}
```

Comment:
I am not a fan of the testing strategy in this PR because:

1. It overrides the plaintext password testcases reducing our coverage
2. It buries `googlesecretmanager` tests in random database app tests. It should really have its own dedicated test cases in the main test framework.

I think this should be redone in a way that addresses both of those points.

Reply:
I agree with this, and I think it should be replaced with an integration test that also creates the secrets in the current project, so the test is not hardcoded to a single project.

Reply:
Done